velox/common/memory/MmapAllocator.cpp

```diff
@@ -593,7 +593,7 @@ int32_t MmapAllocator::SizeClass::findMappedFreeGroup() {
     index = 0;
   }
   auto lookupSize = mappedFreeLookup_.size() + kSimdTail;
-  for (auto counter = 0; counter <= lookupSize; ++counter) {
+  for (auto counter = 0; counter <= lookupSize; counter += kWidth) {
```

Comment:
Would you update the PR description to explain the problem and the solution in some detail?

Reply:
> Would you update the PR description to explain the problem and the solution in some detail?

Updated, thanks for you reply.